fcs/build.fsx

```diff
@@ -24,7 +24,14 @@ let isMono = false
 // Utilities
 // --------------------------------------------------------------------------------------
 
-let dotnetExePath = DotNetCli.InstallDotNetSDK "2.1.403"
+let dotnetExePath =
+    // Build.cmd normally downloads a dotnet cli to: <repo-root>\artifacts\toolset\dotnet
```

Comment:
Does `build.sh` do this as well?

Also when @brettfo finishes up the arcade work it won't be build.cmd/build.sh anymore

Reply:
@cartermp build.sh in fcs calls build.fsx :-)
and build.sh in root builds a makefile.